synapse/tests/test_axon.py

```diff
@@ -177,52 +370,170 @@ def test_axon_clustered(self):
         dmon.share('axons', s_service.SvcBus(), fini=True)
 
         with self.getTestDir() as datadir:
-
             dir0 = gendir(datadir, 'host0')
             dir1 = gendir(datadir, 'host1')
             dir2 = gendir(datadir, 'host2')
+            dir3 = gendir(datadir, 'host3')
+
+            host0 = s_axon.AxonHost(dir0, **{'axonhost:hostname': 'host0',
+                                             'axonhost:axonbus': busurl,
+                                             'axonhost:clones': 1,
+                                             'axonhost:autorun': 2,
+                                             'axonhost:bytemax': s_axon.megabyte * 100,
+                                             'axonhost:syncmax': s_axon.megabyte * 10})
+            host1 = s_axon.AxonHost(dir1, **{'axonhost:hostname': 'host1',
+                                             'axonhost:axonbus': busurl,
+                                             'axonhost:clones': 1,
+                                             'axonhost:autorun': 2,
+                                             'axonhost:bytemax': s_axon.megabyte * 100,
+                                             'axonhost:syncmax': s_axon.megabyte * 10})
+            host2 = s_axon.AxonHost(dir2, **{'axonhost:hostname': 'host2',
+                                             'axonhost:axonbus': busurl,
+                                             'axonhost:clones': 1,
+                                             'axonhost:autorun': 2,
+                                             'axonhost:bytemax': s_axon.megabyte * 100,
+                                             'axonhost:syncmax': s_axon.megabyte * 10})
+            host3 = s_axon.AxonHost(dir3, **{'axonhost:hostname': 'host3',
+                                             'axonhost:axonbus': busurl,
+                                             'axonhost:clones': 1,
+                                             'axonhost:autorun': 2,
+                                             'axonhost:bytemax': s_axon.megabyte * 100,
+                                             'axonhost:syncmax': s_axon.megabyte * 10})
+
+            time.sleep(3)
```

Comment:
can we replace this time.sleep with another check to see how many axons we have?